admin/projects.go

```diff
@@ -90,6 +90,7 @@ func (s *Service) CreateProject(ctx context.Context, opts *database.InsertProjec
 	return res, nil
 }
 
+// Create a new meth for teardown
```

Comment:
Don't understand this comment

Reply:
was just added as todo initially, removing it :)